lib/WTSI/NPG/HTS/Illumina/LogPublisher.pm

```diff
@@ -26,6 +27,8 @@ our $VERSION = '';
 our $DEFAULT_ROOT_COLL = '/seq';
 our $DEFAULT_LOG_COLL  = 'log';
 
+Readonly::Scalar my $BAM_BASECALLS_DEPTH => 4;
```

Comment:
To be consistend, please update definitions of constants om lines 27-28 to

Readonly::Scalar my $DEFAULT_ROOT_COLL => '/seq';
Readonly::Scalar my $DEFAULT_LOG_COLL  => 'log';

Reply:
Should I leave `our` keyword for these variables?

Reply:
No, they are not used anywhere. You can check by searching the code of this packege. Classess from this package are not used anywhere else in NPG code.